sdk/lib/crt/process/_system.c

```diff
@@ -118,6 +119,99 @@ int system(const char *command)
 
 int CDECL _wsystem(const wchar_t* cmd)
 {
-    FIXME("_wsystem stub\n");
-    return -1;
+    wchar_t *szCmdLine = NULL;
+    wchar_t *szComSpec = NULL;
+
+    PROCESS_INFORMATION ProcessInformation;
+    STARTUPINFOW StartupInfo;
+    wchar_t *s;
+    BOOL result;
+
+    int nStatus;
+
+    szComSpec = _wgetenv(L"COMSPEC");
+
+    // system should return 0 if cmd is null and the shell is found
+
+    if (cmd == NULL)
+    {
+        if (szComSpec == NULL)
+            return 0;
+        else
+            return 1;
+    }
+
+    if (szComSpec == NULL)
+        return -1;
+
+    // should return 127 or 0 ( MS ) if the shell is not found
+    // _set_errno(ENOENT);
+
+    if (szComSpec == NULL)
```

Comment:
This can never be true, as you do check that a few lines above already and return then. (spotted by ChatGPT)

Reply:
Fixed.